### PR TITLE
Added Indonesian Translations of CONTRIBUTING.md and CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Bryzizzle**
Added Indonesian Translations of CONTRIBUTING.md and CODE_OF_CONDUCT.md
Additionally, fixed the Indonesian Language code from IN to ID in both the Main README.md and the folder name.

### CODE REVIEW

**Comment by Cyclic3** [comment]
Thanks for your work on this!

**Comment by brandonskerritt** [comment]
This PR is broken slightly:

The URL for ID was not changed, but the folder name was :) 
https://github.com/Ciphey/Ciphey/tree/master/translations/in/README.md

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
 Translations <br>
-<a href=https://github.com/Ciphey/Ciphey/tree/master/translations/in/README.md>🇮🇩In</a>
+<a href=https://github.com/Ciphey/Ciphey/tree/master/translations/in/README.md>🇮🇩 ID</a>
```
Comment (brandonskerritt) [inline README.md:3]: The URL did not change, which makes this link broken in the README

Comment (Bryzizzle) [reply README.md:3]: My bad, I missed that. I see that you've added another pull request so I suppose it should be fixed soon :)

**Comment by Bryzizzle** [comment]
@Cyclic3 Happy to help!

**Comment by bee-san** [comment]
Hey we're changing from MIT to GPLv3 because some of our projects are licensed with that, do you approve? If you do not approve we will be forced to delete your code and rewrite it 😢 

By approving of this change, you are agreeing to re-license the code you wrote under GPLv3.

**Pull request merged**